client/src/components/dashboard/Poll.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
this currently fails when ran, still working on it?

Reply:
fixed issue